js/ui/data_grid.d.ts

```diff
@@ -114,103 +518,18 @@ export interface GridBaseOptions<T = GridBase> extends WidgetOptions<T> {
     columnAutoWidth?: boolean;
     /**
      * @docid
+     * @type object
      * @prevFileNamespace DevExpress.ui
      * @public
      */
-    columnChooser?: {
-    /**
-       * @docid
-     * @prevFileNamespace DevExpress.ui
-       * @default false
-       */
-      allowSearch?: boolean,
-      /**
-       * @docid
-       * @prevFileNamespace DevExpress.ui
-       * @default "Drag a column here to hide it"
-       */
-      emptyPanelText?: string,
-      /**
-       * @docid
-       * @prevFileNamespace DevExpress.ui
-       * @default false
-       */
-      enabled?: boolean,
-      /**
-       * @docid
-       * @prevFileNamespace DevExpress.ui
-       * @default 260
-       */
-      height?: number,
-      /**
-       * @docid
-       * @prevFileNamespace DevExpress.ui
-       * @type Enums.GridColumnChooserMode
-       * @default "dragAndDrop"
-       */
-      mode?: 'dragAndDrop' | 'select',
-      /**
-       * @docid
-       * @prevFileNamespace DevExpress.ui
-       * @default 500
-       */
-      searchTimeout?: number,
-      /**
-       * @docid
-       * @prevFileNamespace DevExpress.ui
-       * @default "Column Chooser"
-       */
-      title?: string,
-      /**
-       * @docid
-       * @prevFileNamespace DevExpress.ui
-       * @default 250
-       */
-      width?: number
-    };
+    columnChooser?: ColumnChooser;
     /**
      * @docid
+     * @type object
      * @prevFileNamespace DevExpress.ui
      * @public
      */
-    columnFixing?: {
-    /**
-      * @docid
-       * @prevFileNamespace DevExpress.ui
-     * @default false
-      */
-      enabled?: boolean,
-      /**
-       * @docid
-     * @prevFileNamespace DevExpress.ui
-       */
-      texts?: {
-        /**
-         * @docid
-         * @prevFileNamespace DevExpress.ui
-         * @default "Fix"
-         */
-        fix?: string,
-        /**
-         * @docid
-         * @prevFileNamespace DevExpress.ui
-         * @default "To the left"
-         */
-        leftPosition?: string,
-        /**
-         * @docid
-         * @prevFileNamespace DevExpress.ui
-         * @default "To the right"
-         */
-        rightPosition?: string,
-        /**
-         * @docid
-         * @prevFileNamespace DevExpress.ui
-         * @default "Unfix"
-         */
-        unfix?: string
-      }
-    };
+    columnFixing?: GridBaseColumnFixing;
```

Comment:
Rename GridBaseColumnFixing to ColumnFixing

Reply:
done